src/notation/view/widgets/editstyle.cpp

```diff
@@ -862,8 +862,8 @@ EditStyle::EditStyle(QWidget* parent)
             connect(checkBox, &QCheckBox::stateChanged, setSignalMapper, mapFunction);
         } else if (auto button = qobject_cast<QAbstractButton*>(sw.widget)) {
             connect(button, &QAbstractButton::toggled, setSignalMapper, mapFunction);
-        } else if (auto groupBox = qobject_cast<QGroupBox*>(sw.widget)) {
-            connect(groupBox, &QGroupBox::toggled, setSignalMapper, mapFunction);
+        } else if (auto myGroupBox = qobject_cast<QGroupBox*>(sw.widget)) {
```

Comment:
why is it needed? didn't see the same name above

Reply:
It is there, somewhere, check the output of other Windowes builds:
```
2023-05-09T13:35:01.2361479Z D:/a/MuseScore/MuseScore/src/notation/view/widgets/editstyle.cpp(865): warning C4458: declaration of 'groupBox' hides class member
2023-05-09T13:35:01.4484152Z D:\a\MuseScore\MuseScore\build.release\src\notation\ui_editstyle.h(1017): note: see declaration of 'Ui_EditStyleBase::groupBox'
```

Reply:
Maybe we can fix it there, will have a look.